readme.md

```diff
@@ -135,41 +142,66 @@ var a = 2;
 
 ### Default parameters (5 / 40)
 
-With ES6, we now have the option add set a default value for any of our functions' parameters.
+If a function defined to take 3 parameters is called with two arguments, what is the value of the third parameter in the function body?
+
+For example:
+
+```js
+function printAndSum(a, b, c){
+console.log(a)
+console.log(b)
+console.log(c)
+
+return a + b + c
+}
+
+printAndSum(1, 2)
+
+// => ???
+```
+
+With ES6, we now have the option to set a default value for any of our functions' parameters.
 
 ```js
 function hello( name = "stranger"){
```

Comment:
I the interest of continuity I think this ES6 example function should be an updated version of the prior ES5 function. Either by promoting `hello` to an ES5 version above (and altering the preamble text accordingly), or replacing `hello` here with the ES6 "parameter defaults" version of `printAndSum`